### DISCUSSION TITLE
Custom CSS selector when tests are running

### DISCUSSION THREAD

**Discussion by d3vl1m3** [oblador/loki#430] [Q&A]
Does Loki add a custom selector for when tests are running? I need to mask some images that we don't have full control over.

I know I can use `isLokiRunning()` and apply a class based on that but wondered if there was one added automagically :)

**Comment by abhishekZom** [comment]
yes it does. The custom selector is [data-loki], and it is added to the body element of the page.

You can use this custom selector to apply styles to the page while tests are running, or to detect when tests are running and take certain actions
